lib/subproviders/geth_api_double.js

```diff
@@ -519,4 +519,301 @@ GethApiDouble.prototype.debug_traceTransaction = function(tx_hash, params, callb
+  callback(null, false);
+};
+
+/**
+ * Used for submitting mining hashrate.
+ *
+ * @param {String} hashRate - a hexadecimal string representation (32 bytes) of the hash rate
+ * @param {String} clientID - A random hexadecimal(32 bytes) ID identifying the client
+ * @callback callback
+ * @param {error} err - Error object
+ * @param {Boolean} result - returns true if submitting went through succesfully and false otherwise.
+ */
+GethApiDouble.prototype.eth_submitHashrate = function(hashRate, clientID, callback) {
+  callback(null, false);
+};
+
+
+/**
+ * Stores a string in the local database.
+ *
+ * @param {String} dbName - Database name.
+ * @param {String} key - Key name.
+ * @param {String} value - String to store.
+ * @callback callback
+ * @param {error} err - Error object
+ * @param {Boolean} result - returns true if the value was stored, otherwise false.
+ */
+GethApiDouble.prototype.db_putString = function(dbName, key, value, callback) {
+  callback(null, false);
+};
+
+/**
+ * Returns string from the local database
+ *
+ * @param {String} dbName - Database name.
+ * @param {String} key - Key name.
+ * @callback callback
+ * @param {error} - Error Object
+ * @param {String} result - The previously stored string.
+ */
+GethApiDouble.prototype.db_getString = function(dbName, key, callback) {
+  callback(null, "");
+};
+
+/**
+ * Stores binary data in the local database.
+ *
+ * @param {String} dbName - Database name.
+ * @param {String} key - Key name.
+ * @param {DATA} data - Data to store.
+ * @callback callback
+ * @param {error} err - Error Object
+ * @param {Boolean} result - returns true if the value was stored, otherwise false.
+ */
+GethApiDouble.prototype.db_putHex = function(dbName, key, data, callback) {
+  callback(null, false);
+};
+
+/**
+ * Returns binary data from the local database
+ *
+ * @param {String} dbName - Database name.
+ * @param {String} key - Key name.
+ * @callback callback
+ * @param {error} err - Error Object
+ * @param {DATA} result - The previously stored data.
+ */
+GethApiDouble.prototype.db_getHex = function(dbName, key, callback) {
+  callback(null, "0x00");
+};
+
+
+/**
+ * Sends a whisper message.
+ *
+ * @param {DATA, 60 Bytes} from - (optional) The identity of the sender.
+ * @param {DATA, 60 Bytes} to - (optional) The identity of the receiver. When present whisper will encrypt the message so that only the receiver can decrypt it.
+ * @param {Array of DATA} topics - Array of DATA topics, for the receiver to identify messages.
+ * @param {DATA} payload - The payload of the message.
+ * @param {QUANTITY} priority - The integer of the priority in a range from ... (?).
+ * @param {QUANTITY} ttl - integer of the time to live in seconds.
+ * @callback callback
+ * @param {error} err - Error Object
+ * @param {Boolean} result - returns true if the message was sent, otherwise false.
+ */
+GethApiDouble.prototype.shh_post = function(from, to, topics, payload, priority, ttl, callback) {
```

Comment:
I feel like the whisper-specific methods should be broken out into their own file, but I wouldn't be opposed to this happening once we add whisper support in the future